src/devices/bus/sg1000_exp/sp400.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+  license:BSD-3-Clause
```

Comment:
This doesn't follow MAME convention for licensing and author headers.
Should be more or less like this on the first two lines, cfr. any file in the codebase:
```
// license: BSD-3-Clause
// copyright-holders: Charles MacDonald, Devin Hill, Fabio Dalla Libera
```

Reply:
thank you, fixed